web/client/plugins/TOC.jsx

```diff
@@ -685,6 +685,7 @@ const checkPluginsEnhancer = branch(
  * @prop {boolean} cfg.activateAddGroupButton: activate a button to add a new group, default `true`
  * @prop {boolean} cfg.showFullTitleOnExpand shows full length title in the legend. default `false`.
  * @prop {boolean} cfg.hideOpacityTooltip hide toolip on opacity sliders
+ * @prop {boolean} cfg.activateRemoveGroup if set to false, do not show the remove button for layer groups. default `true`
```

Comment:
```suggestion
 * @prop {boolean} cfg.activateRemoveGroup if set to false, do not show the remove button for layer groups. default `true`
  * @prop {boolean} [addLayersPermissions=true] if false, only users of role ADMIN can see has the permission to see add layers button. Default true.
   @prop {boolean} [removeLayersPermissions=true] if false, only users of role ADMIN can see has the permission to remove layers. Default true.
   @prop {boolean} [sortingPermissions=true] if false, only users of role ADMIN can see has the permission to move layers in the TOC. Default true.
   @prop {boolean} [addGroupsPermissions=true] if false, only users of role ADMIN can see has the permission add groups to the TOC. Default true.
   @prop {boolean} [removeGroupsPermissions=true] if false, only users of role ADMIN can see has the permission remove groups from the TOC. Default true.
   @prop {boolean} [layerInfoToolPermissions=false] if false, only users of role ADMIN can see has the permission see the layer info tool.. Default false.
```

Reply:
all those lines have `can see has`, i think there's an extra verb ;)

Reply:
Thank you :facepalm: fixed here https://github.com/geosolutions-it/MapStore2/pull/7044